MM/5.8.0/config/media_manager_5/labels/Asset List.dcl

```diff
@@ -1384,7 +1384,7 @@ resource configservice_label asset_list_delete_confirmation_dialog_content {
       language_id = data.language.english.id
     },
     {
-      default_translation = 'Er du sikker på at du vil slette {multiple, select, true {assetsne} false {assettet}} permanent? Denne handling kan ikke fortrydes, og {multiple, select, true {assetsne} false {assettet}} kan ikke gendannes.'
+      default_translation = 'Er du sikker på at du vil slette {multiple, select, true {assets} false {asset}} permanent? Denne handling kan ikke fortrydes, og {multiple, select, true {assetsne} false {assettet}} kan ikke gendannes.'
```

Comment:
```suggestion
      default_translation = 'Er du sikker på at du vil slette {multiple, select, true {assets} false {asset}} permanent? Denne handling kan ikke fortrydes, og {multiple, select, true {assets} false {asset}} kan ikke gendannes.'
```

Reply:
This is ok to change, even though I slightly disagree

Denne handling kan ikke fortrydes, og assetsne kan ikke gendannes.
-->
Denne handling kan ikke fortrydes, og assets kan ikke gendannes.

----------

This is bad grammar imo

Denne handling kan ikke fortrydes, og assettet kan ikke gendannes.
-->
Denne handling kan ikke fortrydes, og asset kan ikke gendannes.